modules/ROOT/pages/clauses/where.adoc

```diff
@@ -486,18 +486,25 @@ The `name` and `age` for `Andy` are returned because his name starts with 'AND'
 [[filter-on-patterns]]
 === Filter on patterns
 
-Patterns are expressions in Cypher, expressions that return a list of paths.
-List expressions are also predicates; an empty list represents `false`, and a non-empty list represents `true`.
-
-Patterns are not only expressions, they are also predicates.
-The only limitation to a pattern is that it must be able to express it in a single path.
-Unlike `MATCH`, it is not possible to use commas between multiple paths.
-To combine multiple patterns using `WHERE`, instead use `AND`.
-Note that new variables cannot be introduced.
-Although it might look very similar to the `MATCH` patterns, the `WHERE` clause is used to eliminate matched paths.
-`MATCH (a)-[*]->(b)` is very different from `WHERE (a)-[*]->(b)`.
-The first will produce a path for every path it can find between `a` and `b`, whereas the latter will eliminate any matched paths where `a` and `b` do not have a directed relationship chain between them.
+Path patterns may be used both in a MATCH clause and in a WHERE clause. When used in a WHERE clause they are called path
```

Comment:
There are more expression positions for path patterns than just the `WHERE` clause:
- `exists(<path pattern>)`
- `CASE WHEN <path pattern> THEN ... END`